pkg/controller/directvolumemigrationprogress/directvolumemigrationprogress_controller.go

```diff
@@ -267,22 +268,35 @@ func (r *ReconcileDirectVolumeMigrationProgress) reportContainerStatus(pvProgres
 		if transferRate != "" {
 			pvProgress.Status.LastObservedTransferRate = transferRate
 		}
+		pvProgress.Status.Runtime = nil
 	case !containerStatus.Ready && containerStatus.LastTerminationState.Terminated != nil && containerStatus.LastTerminationState.Terminated.ExitCode != 0:
 		// pod has a failure, report last failure reason
 		pvProgress.Status.PodPhase = kapi.PodFailed
 		pvProgress.Status.LogMessage = containerStatus.LastTerminationState.Terminated.Message
+		exitCode := containerStatus.LastTerminationState.Terminated.ExitCode
```

Comment:
Does container return the same return code as returned by the startup command?

Reply:
yes